manifests/modules/automation/controlplanes/crossplane/managed/table.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: dynamodb.aws.upbound.io/v1beta1
+kind: Table
+metadata:
+  name: items
+  labels:
+    testing.upbound.io/example-name: dynamodb
+  annotations: 
+    crossplane.io/external-name: items
```

Comment:
This name will have to align with the cluster name, so should be something like `${EKS_CLUSTER_NAME}-carts-crossplane`

Reply:
Updated